api/db/changelog/db.changelog-115-addColumn-user-create-modified.xml

```diff
@@ -7,17 +7,11 @@
   <changeSet author="nsaxena" id="db.changelog-115-addColumn-user-create-modified">
     <addColumn tableName="user">
       <column name="creation_time" type="datetime">
-        <constraints nullable="false" />
+        <constraints nullable="true" />
       </column>
       <column name="last_modified_time" type="datetime">
-        <constraints nullable="false"/>
+        <constraints nullable="true"/>
       </column>
     </addColumn>
   </changeSet>
-  <changeSet id="update creation time" author="nsaxena">
-    <sql>update user set creation_time = first_sign_in_time</sql>
-  </changeSet>
-  <changeSet id="update modified time" author="nsaxena">
-    <sql>update user set last_modified_time = first_sign_in_time</sql>
```

Comment:
I think its OK to leave the last modified time update sql. That value changes often and its not as important to ensure its integrity.